tests/create_all/main.tf

```diff
@@ -0,0 +1,20 @@
+module "state_bucket" {
+  source = "git::https://github.com/plus3it/terraform-aws-tardigrade-s3-bucket.git?ref=5.0.0"
```

Comment:
This isn't testing _this_ module at all. It's just invoking the remote s3 module. Every test config should, at least once, invoke _this_ module using `source = "../../"`